PendingReleaseNotes

```diff
@@ -3,3 +3,41 @@
 * RGW's default backend for `rgw_enable_ops_log` changed from RADOS to file.
   The default value of `rgw_ops_log_rados` is now false, and `rgw_ops_log_file_path`
   defaults to "/var/log/ceph/ops-log-$cluster-$name.log".
+* RGW's pubsub interface now returns boolean fields using bool. Before this change,
```

Comment:
Should `bool` be \`\`bool\`\` ?

Reply:
in terms of explanation above, how about replacing "bool" with `"true"` or `"false"`. it's probably kind of pedantic though. but more correct, IMHO. what do you think?

Reply:
Agreed

Reply:
not sure about that; every language binding for JSON that I know of will convert 'true' or 'false' to a boolean of the appropriate state, and while JSON doesn't "define" it, the bindings do.  Also, they're more than just string literals; they're distinguished in the spec as one of three "literal name tokens" ("null" being the third).  I think they're as close to boolean as any representation without an actual type definition could possibly be.

Some online references actually call them booleans, too (although I agree, pedantically, incorrectly).  The reason isn't so much that there isn't a boolean; the reason is that more that there are no variables for which to restrict the possible values.

And Javascript has booleans.

Reply:
I raised my initial question because I'm not that familar with the language / API in question, so I wasn't sure if `bool` here is short for `boolean`, if it's a specific literal, etc.